### PR TITLE
File's Categories are now nullable

### PR DESCRIPTION
**Pull request by KevinBoulongne**
[Sentry: https://sentry.infomaniak.com/organizations/infomaniak/issues/12305/?project=3](https://sentry.infomaniak.com/organizations/infomaniak/issues/12305/?project=3)

### CODE REVIEW

File: app/src/main/java/com/infomaniak/drive/data/models/File.kt [hunk 1]
```diff
@@ -101,7 +101,7 @@ open class File(
     var type: String = "file",
     var users: @WriteWith<IntRealmListParceler> RealmList<Int> = RealmList(),
     var visibility: String = "",
-    var categories: @RawValue RealmList<FileCategory> = RealmList(),
+    var categories: @RawValue RealmList<FileCategory>? = RealmList(),
```
Comment (JorisBodin) [inline app/src/main/java/com/infomaniak/drive/data/models/File.kt:104]: No need for migration?

Comment (sirambd) [reply app/src/main/java/com/infomaniak/drive/data/models/File.kt:104]: Yes migration is needed here

**Review by sirambd — CHANGES_REQUESTED** [review]
waiting for request changes

**Comment by KevinBoulongne** [comment]
Waiting for API v2 to test it again.

**Pull request closed without merging**